Task 3/Task 3.3/Task 3.3/Task 3.3/ExtensionMethods.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace Task_3._3
+{
+    public static class ExtensionMethods
+    {
+        public static void CheckLanguage(this string text)
+        {
+            Regex english = new Regex(@"^\W*([a-zA-z]+\W*\s*)+$");
+            Regex russian = new Regex(@"^\W*([а-яёА-ЯЁ]+\W*\s*)+$");
+            Regex digits = new Regex(@"^\W*([0-9]+\W*\s*)+$");
```

Comment:
* By task punctuation and spaces in text can make it mixed
* since `\w` is `[A-Za-z0-9_]`, english regex will be okay with `лолenglish ` and others with `虾米`